packages/toaster/stories/Toaster.stories.tsx

```diff
@@ -45,6 +45,12 @@ export const Warning = () => (
   </Toaster>
 );
 
+export const Mute = () => (
```

Comment:
@nataliepina do you know if there is a way to combine all of these appearance stories into one and just let a user change the appearance in Storybook?

Reply:
Yes @vacas5, you can combine all of the appearances by using controls similar to the Badge component story - https://dcos-labs.github.io/ui-kit/?path=/story/graphic-elements-badge--default